packages/blocks/tests/php/test-blocks.php

```diff
@@ -84,4 +111,116 @@ public function test_is_amp_request() {
 	public function test_is_not_amp_request() {
 		$this->assertFalse( Blocks::is_amp_request() );
 	}
+
+	/**
+	 * Test WordPress and Gutenberg version requirements.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::is_gutenberg_version_available
+	 */
+	public function test_returns_false_if_core_wp_version_less_than_minimum() {
+		$version_gated = Blocks::is_gutenberg_version_available(
+			array(
+				'wp'        => '999999',
+				'gutenberg' => '999999',
+			),
+			'gated_block'
+		);
+		$this->assertEquals( false, $version_gated );
+	}
+
+	/**
+	 * Test WordPress and Gutenberg version requirements.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::is_gutenberg_version_available
+	 */
+	public function test_returns_true_if_core_wp_version_greater_or_equal_to_minimum() {
+		$version_gated = Blocks::is_gutenberg_version_available(
+			array(
+				'wp'        => '1',
+				'gutenberg' => '999999',
+			),
+			'ungated_block'
+		);
+		$this->assertEquals( true, $version_gated );
+	}
+
+	/**
+	 * Testing removing the Jetpack prefix from a block slug.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::remove_extension_prefix
+	 *
+	 * @dataProvider get_extension_name_provider
+	 *
+	 * @param string $extension_slug      Block / Extension name.
+	 * @param string $expected_short_slug Extension name without Jetpack prefix.
+	 */
+	public function test_remove_extension_prefix( $extension_slug, $expected_short_slug ) {
+		$short_slug = Blocks::remove_extension_prefix( $extension_slug );
+
+		$this->assertEquals( $short_slug, $expected_short_slug );
+	}
+
+	/**
+	 * Get different possible block names.
+	 *
+	 * Data provider for test_remove_extension_prefix.
+	 */
+	public function get_extension_name_provider() {
+		return array(
+			'not_jetpack'    => array(
+				'woocommerce/product-best-sellers',
+				'woocommerce/product-best-sellers',
+			),
+			'jetpack_dash'   => array(
+				'jetpack/shortlinks',
+				'shortlinks',
+			),
+			'jetpack_hyphen' => array(
+				'jetpack-shortlinks',
+				'shortlinks',
+			),
+		);
+	}
+
+	/**
+	 * Test to ensure that an extension is returned as registered.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::is_registered
+	 */
+	public function test_is_extension_registered() {
+		// Test for the block that is registered for all tests here.
+		$this->assertTrue( Blocks::is_registered( $this->block_name ) );
+		// Test for a non-existing block.
+		$this->assertFalse( Blocks::is_registered( 'foo/bar' ) );
+	}
+
+	/**
+	 * Test to ensure blocks are registered.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::jetpack_register_block
+	 */
+	public function test_jetpack_register_block() {
+		$this->assertEquals( 'jetpack/apple', $this->block_name );
```

Comment:
This test doesn't seem to actually test anything? Except maybe the test class itself.

Reply:
You're right. Removed in 7a22cd0f033f51312c30edbdcb5d4256c0b7569d